### ISSUE TITLE
scale and aggregate the connectivity indicator

### ISSUE THREAD

**Issue by anders-kolstad** [NINAnor/ecosystemCondition#144]
The connectivity indicator is not yet complete and is lacking two key steps. 

Currently there are two shape files, one with the reference values (connectivity without infrastructure) and one represneting the the current condition with infrastructure To normalise (i.e. scale) the current condition we need to take the quotient. Secondly, we need to aggregate the scaled indicator values to each of the five regions in Norway. In this step we can take the spatial variation, the SD, and use that as the uncratinty for the real regional indicator values.

**Comment by VegarBakkestuen** [comment]
Jeg er usikker på om jeg har riktig shapefil for de 5 regionene i Norge. Kan du henvise til hvor de ligger. Jeg fant det ikke på Sharepoint nå.

**Comment by anders-kolstad** [comment]
De ligger i [`data/`](https://github.com/NINAnor/ecosystemCondition/tree/main/data) på github og heter _regions_

**Comment by VegarBakkestuen** [comment]
![image](https://github.com/NINAnor/ecosystemCondition/assets/58920522/37e1c42a-7184-46e2-a964-30eb705fac21)

The data are very big and I am currently downloading from Google Earth Engine. I hope to be finished by tomorrow.

**Comment by VegarBakkestuen** [comment]
Basicly, can I do this so simple to the mean of the connectivity kvotsient for each of the five regions?

**Comment by VegarBakkestuen** [comment]
Here is the average (mean) value of the connectivity for each region:

Region 1 Nord-Norge 0,174110748
Region 2 Midt-Norge 0,108677
Region 3 Østlandet 0,299318
Region 4 Vestlandet 0,068484
Region 5 Sørlandet 0,222424

**Comment by anders-kolstad** [comment]
Yes, the mean is probably good. No area weighting. But don't these numbers look a bit strange? I expected realtively higher numbers (better condition) in N-Norway for example, and higher numbers overall. This is the mean of the scaled indikcator righrt? I.e. the mea of the quotients?

**Comment by VegarBakkestuen** [comment]
Here are the new numbers from the new script:

Region 1 Nord-Norge 0,89631698
Region 2 Midt-Norge 0,705864078
Region 3 Østlandet 0,518705783
Region 4 Vestlandet 0,629798357
Region 5 Sørlandet 0,663023587
